addons/code-analysis/style.css

```diff
@@ -0,0 +1,239 @@
+.ps-container {
+  position: relative;
+  background: #393943;
+  width: 960px;
+  height: 283px;
+  border-radius: 13px;
+}
+
+.ps-header {
+  position: absolute;
+  left: 15px;
+  top: 12px;
+  background: #ffffff;
+  width: 159px;
+  height: 44px;
+  border-radius: 13px;
+}
+
+.ps-header-right {
+  position: absolute;
+  float: right;
+  right: 47px;
+  background: #ffffff;
+  width: 144px;
+  height: 39px;
+  border-radius: 9px;
+}
+
+.ps-header-right-content {
+  position: absolute;
+  left: 15px;
+  top: 9px;
+  font-size: 1rem;
+  font-weight: bold;
+}
+
+.ps-header-content {
+  position: absolute;
+  left: 14px;
+  top: 5px;
+  color: #393943;
+  font-size: 1rem;
+  font-weight: bold;
+}
+
+.ps-content {
+  position: absolute;
+  top: 36px;
+  left: 15px;
+  background: #ffffff;
+  width: 930px;
+  height: 233px;
+  border-radius: 13px;
+}
+
+[class^="ps-edge"] {
+  position: absolute;
+  top: 11%;
+  width: 50px;
+  height: 80px;
+}
+
+.ps-edge01 {
+  left: 2%;
+}
+
+.ps-edge02 {
+  left: 92.33%;
+  transform: rotateY(180deg);
+}
+
+.ps-code-genetic {
+  position: relative;
+  height: 100px;
+}
+
+.ps-code-genetic-bg {
+  position: absolute;
+  background: #575757;
+  top: 14%;
+  left: 5%;
+  width: 90%;
+  height: 68px;
+}
+
+span.motion {
+  background-color: #548af6;
+}
+
+span.looks {
+  background-color: #865df3;
+}
+
+span.sound {
+  background-color: #c059ca;
+}
+
+span.events {
+  background-color: #f6d13f;
+}
+
+span.control {
+  background-color: #f0a43b;
+}
+
+span.sensing {
+  background-color: #62b5de;
+}
+
+span.operators {
+  background-color: #5cb550;
+}
+
+span.variables {
+  background-color: #ee8734;
+}
+
+span.my-blocks {
+  background-color: #e86577;
+}
+
+[class^="ps-code-genetic-block"] {
+  position: absolute;
+  top: 14%;
+  height: 68px;
+}
+
+.ps-code-genetic-table {
+  position: relative;
+  height: 100px;
+}
+
+.ps-code-genetic-table > [class^="ps-code-genetic-item"] {
+  position: absolute;
+  font-size: 1rem;
+  font-weight: bold;
+}
+
+span.row0 {
+  top: 1px;
+}
+
+span.row1 {
+  top: 31px;
+}
+
+span.row2 {
+  top: 62px;
+}
+
+span.col0 {
+  left: 201px;
+}
+
+span.col1 {
+  left: 390px;
+}
+
+span.col2 {
+  left: 601px;
+}
+
+span.col3 {
+  left: 802px;
+}
+
+.ps-code-genetic-item-motion {
+  left: 89px;
+  color: #548af6;
+  text-transform: capitalize;
```

Comment:
Could this be added to `.ps-code-genetic-table` so it's not repeated so many times.